source/HiringPanel.cpp

```diff
@@ -107,12 +107,12 @@ bool HiringPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, b
 	if(!player.Flagship())
 		return false;
 	
-	if(key == 'h' || key == '=' || key == SDLK_RETURN || key == SDLK_SPACE)
+	if(key == 'h' || key == SDLK_EQUALS || key == SDLK_KP_PLUS || key == SDLK_RETURN || key == SDLK_SPACE)
```

Comment:
#5624 replaced `key == '='` with `key == SDLK_PLUS || key == SDLK_KP_PLUS || key == SDLK_EQUALS`. Here you've dropped `key == SDLK_PLUS`. Shouldn't that be used here as well? (And in the trading panel instance.)

Reply:
I'm not sure. I don't think that the game registers the SDLK_PLUS key, because it should only work with shift + '='.